qt/widgets/mplcpp/src/ColorbarWidget.cpp

```diff
@@ -50,6 +51,7 @@ ColorbarWidget::ColorbarWidget(QWidget *parent)
     : QWidget(parent), m_ui(), m_mappable(Normalize(0, 1), getCMap(defaultCMapName())) {
   initLayout();
   connectSignals();
+  m_canvas->installEventFilterToMplCanvas(new FigureEventFilter());
```

Comment:
Do you know what happens to the ownership of the new event filter? I see it gets passed through to QWidget::installEventFilter but I can't tell whether the QWidget takes ownership of it at that point so I'm concerned it might leak?

Reply:
Good point, I've opened a PR to fix this https://github.com/mantidproject/mantid/pull/36204